docs/experience_date_search/index.md

```diff
@@ -45,6 +45,52 @@ You can enable customers to move between the results pages using the `paginate`
 
 You can assign the value of a Liquid variable to `page_size`.
 
+## Available Months
+The experience_date_search tag returns `available_months`, an array of `Date` objects, which can be used to construct the month and year filters for searching.
+
+Each date object is dated 1st of the month, and there is one for each month in which a product's slots start_on dates occur.
+
+E.g. Today is 1st Jan 2024. You have a product with a 10 day duration, with the following slots (all future or ongoing);
+31th Dec 2023
```

Comment:
```suggestion
31th Dec 2023
```

suggestion/question: are this and the below dates indended to be formatted as a list? If so, think each line will need to start with a `- `.

### Currently
![Screenshot 2023-08-15 at 17 46 24](https://github.com/easolhq/easolhq.github.io/assets/12126304/535cc09b-cbac-495f-a51b-b07adc87f0ed)

### Suggestion
![Screenshot 2023-08-15 at 17 45 55](https://github.com/easolhq/easolhq.github.io/assets/12126304/983ae3ab-24de-4361-a041-598ba23146e6)

Reply:
Oof, embarrassingly I did load and look at this btw. Somehow saw nothing afoot, I must be broken. Thank you 🖤